### PR TITLE
add DesignConcept.md and svg images

### PR DESCRIPTION
**Pull request by katahiromz**

### CODE REVIEW

**Comment by katahiromz** [comment]
You can open and edit the svg files by Inkscape.

File: svg/start.svg [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (sagudev) [inline svg/start.svg]: Green start menu don't have black boarder

**Comment by sagudev** [comment]
Good start.

**Comment by sagudev** [comment]
Can you please open pull request to other branch

**Pull request closed without merging**